### DISCUSSION TITLE
Updating to the newest release

### DISCUSSION THREAD

**Discussion by luuktimmermans** [upptime/upptime#837] [General] [answered]
Hi,

Currently I’m running Upptime version 1.30.0, but I just found out that the newer version 1.31.0 released a couple of days ago. How I would I update mine to the newest version? I’ve already tried running the Update Template CI action manually, without result.

Anny suggestions?

**Comment by AnandChowdhary** [comment]
You can run the "Update Template CI" manually and it'll update to the latest version! I have to trigger it on the main repository once before people are able to do it, which I've done now :)

**Reply by luuktimmermans** [reply]
I just ran the “Uptime Template CI” action manually, but it’s still not updated to the newest version. What am I doing wrong here?

**Reply by AnandChowdhary** [reply]
Can you share a link to your repository?

**Reply by luuktimmermans** [reply]
Sure! It's https://github.com/lincgit/linc-uptime-monitor

**Reply by AnandChowdhary** [reply]
<img width="848" alt="CleanShot 2023-08-22 at 09 53 00@2x" src="https://github.com/upptime/upptime/assets/2841780/92646621-fa2f-481e-b187-5f9b96ae73fc">

I can see from your GitHub Actions logging (https://github.com/lincgit/linc-uptime-monitor/actions/runs/5933084370/job/16087985814) that your access token does not have the "workflow" permissions required to update the template. Could you please check?

**Reply by luuktimmermans** [reply, marked as answer]
Hi, I just found the solution. I was still using the legacy personal access token. Appears that's not working anymore!
